### PR TITLE
Fix For #1729.

### PR DESCRIPTION
**Pull request by DeepakRajendrakumaran**

### CODE REVIEW

File: src/opt.cpp [hunk 1]
```diff
@@ -295,25 +295,38 @@ static uint64_t lConstElementsToMask(const llvm::SmallVector<llvm::Constant *, I
...
         }
         // Is the high-bit set?  If so, OR in the appropriate bit in
         // the result mask
         if (intMaskValue.countLeadingOnes() > 0)
             mask |= (1ull << i);
     }
+
+    // if no bits are set in mask, do not need to consider undefs. It's
+    // always 'all_off'.
+    // If any bits are set in mask, assume' undef' bits as as '1'. This ensures
```
Comment (dbabokin) [inline src/opt.cpp:325]:
```suggestion
    // If any bits are set in mask, assume 'undef' bits as '1'. This ensures
```

**Comment by DeepakRajendrakumaran** [comment]
Got merged. Will change comment as part of next fix

**Comment by cloudhan** [comment]
I'd suggest you use 
```
Fix #1729 - blah blah
```
See https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword for more info.

**Comment by dbabokin** [comment]
@cloudhan Thanks for the link, good idea.

@DeepakRajendrakumaran @aneshlya FYI.

**Pull request merged**